jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleRawConfiguration.java

```diff
@@ -136,4 +143,14 @@ public boolean getUseOnlyProjectCache() {
   public ImageFormat getImageFormat() {
     return jibExtension.getContainer().getFormat();
   }
+
+  @Override
+  public Optional<Path> getDockerClientExecutable() {
```

Comment:
Just an alternative suggestion: I'm thinking maybe since these are only used when calling `processCommonConfigurationForDockerDaemonImage`, perhaps we can just have them be passed in via parameters to `processCommonConfigurationForDockerDaemonImage` like how the `tarImagePath` is a parameter to `processCommonConfigurationForTarImage`, rather than add them to the `RawConfiguration ?

Reply:
Thought about that too, and since you said it, let's do it that way.